modules/templates/stock-static.xml

```diff
@@ -26,6 +26,9 @@
         <showIn>none</showIn>
         <title>Stocks Custom HTML</title>
         <properties>
+            <property id="templateType" type="hidden">
```

Comment:
A hidden property which changes the behaviour of a different field doesn't seem right to me.

Reply:
At the moment, this is the way I can identify that it's coming from the static template or not.

Reply:
Why does that matter?

Reply:
The behavior of the script for autosuggestion won't work for properly for both element and static template. If you see my changes on `xibo-cms.js` there are configuration that works for element but not for static template and vice versa. I can't explain it yet why it does that